src/ServerClient/ADXClient.ts

```diff
@@ -69,7 +71,9 @@ export class ADXClient {
         "Content-Type": "application/json",
         Authorization: `Bearer ${token}`,
         "x-ms-client-request-id": `KTrender;${self.crypto.randomUUID()}`,
-        "x-ms-app": "KustoTrender"
+        "x-ms-app": "KustoTrender",
+        "x-ms-client-version": packageJson.version,
```

Comment:
Does the JS API also have the Pair where we pass K-V like in Java. Do you think it makes sense to capture the UserAgent (perhaps debug some JS issue? )